docs/src/Box.doc.js

```diff
@@ -171,10 +171,9 @@ card(
         href: 'absolutePositioning',
       },
       {
-        name: 'shape',
-        type: `"square" | "rounded" | "pill" | "circle"`,
-        defaultValue: 'square',
-        href: 'shapes',
+        name: 'rounding',
+        type: `"pill" | "circle" | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8`,
```

Comment:
Why not use one of these scales?
1) 0-12: This makes sense for `padding` and `margin` but for radius we aren't actually using values higher than 8 boints for radius unless it's a `circle` or `pill` type use case.
2) xs-xl: There's additional mental overhead to convert a pixel value to a tshirt size value. When looking at a mockup for a design it's easier to do the math of dividing by 4 than needing to look at the docs to remember which size maps to which pixel value.